app/assets/stylesheets/custom.scss

```diff
@@ -217,21 +214,21 @@ ul, menu, dir {
 }
 
 @media only screen and (max-width: 1000px) {
-      ul.slides li {
-        width: 33%; } }
-    @media only screen and (max-width: 767px) {
-      ul.slides li {
-        width: 50%; } }
+    ul.slides li {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Avoid qualifying class selectors with an element.